Jenkinsfile.cloud

```diff
@@ -69,4 +73,28 @@ node(env.NODE) {
             """
         }
     }
+
+    stage("Create AMI") {
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+            // not packaged yet, just build from src for now
+            sh "dnf install -y git golang"
+            sh "git clone https://github.com/coreos/mantle"
+            sh "mantle/build ore"
+
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
+                              credentialsId: env['AWS_CREDENTIALS']]]) {
+                sh """
+                    # use a symlink so that our uploaded filename is unique
+                    ln -sf ${output}/cloud/latest/rhcos.vmdk rhcos-${commit}.vmdk
+                    mantle/bin/ore aws upload --region us-east-1 \
+                        --ami-name 'rhcos_dev_${commit[0..6]}' \
```

Comment:
The pruner looks at the **Tag** `Name`. Can `ore` take an argument to set the tag as well?

Reply:
I took a quick look and it doesn't look like it uses the tag today:

- https://github.com/coreos/mantle/blob/master/cmd/ore/aws/aws.go#L50-L55
- https://github.com/coreos/mantle/blob/master/cli/cli.go#L55-L60

Reply:
Currently, mantle's tagging is pretty hardcoded, but we can add functionality as needed.  How does the pruner use the `Name` tag?

Reply:
@bgilbert can you verify?

Reply:
@dobbymoodge :arrow_up:

Reply:
@bgilbert It matches against the Name tag as a first step in classifying the AMI. Very roughly, something like `AMI_WILDCARDS["rhcos_dev_*"] = {:keep => 5}` means to deregister AMIs prefixed with "`rhcos_dev_`", except the 5 most recent.

Reply:
That sounds like it would work to me.

Reply:
@jlebon Ah yeah, you're right. That was an extended brain fart I had there, sorry. Regardless, the more sophisticated pruning logic **does** key off of the tags, and that's the direction we have planned to go in for future changes, so that mantle PR makes sense. Thanks, and sorry about the confusion.